src/test/java/com/boardgo/unittest/user/UserInfoEntityTest.java

```diff
@@ -126,4 +128,16 @@ public class UserInfoEntityTest {
         // then
         assertThat(userInfoEntity.getUserInfoStatus().getPushToken()).isEqualTo(pushToken);
     }
+
+    @Test
+    @DisplayName("유저 삭제 처리할 수 있다")
+    void 유저_삭제_처리할_수_있다() {
+        // given
+        LocalDateTime now = LocalDateTime.now();
+        UserInfoEntity userInfoEntity = localUserInfoEntity();
+        // when
+        userInfoEntity.delete(now);
+        // then
+        assertThat(userInfoEntity.getDeleteAt()).isEqualTo(now);
```

Comment:
혹시 `LocalDateTime.now();` 에 대한 테스트가 이 부분 맞을까요?
매개변수를 전달하지 않도록 변경한다면 이렇게 테스트 변경하는건 어떤가요?!

```suggestion
        // then
        assertThat(userInfoEntity.getDeleteAt()).isBefore(LocalDateTime.now());
```

Reply:
이렇게 해도 다를거에요. Update된 시점하고 테스트하는 시점은 밀리초 단위로든 초 단위로든 다를테니까요!

Reply:
음.. 탈퇴 후, 탈퇴 시점이 현재 시점보다 이전이기만 하면 테스트 검증이 되는거니까 달라도 되지 않나요?

Reply:
근데 다시 생각해보니 너무나 빠른 컴퓨터면 정말 같을수도 있어서 실패할 수도 있지 않을까요? 
만에 하나라도 실패하지 않고 명확하게 테스트할 수 있는게 더 좋은 코드 아닐까 생각합니다 ㅎㅎ

Reply:
그럼 `isBeforeOrEqualTo(LocalDateTime.now())` 도 있어요!

Reply:
이거는 가독성의 문제니깐 일단 반영해도 괜찮을까요!?